xtask/src/runchecks.rs

```diff
@@ -289,9 +280,6 @@ fn std_checks() {
 }
 
 fn check_typos() {
-    // Install typos-cli
-    cargo_install(["typos-cli", "--version", "1.16.5"].into());
```

Comment:
Does that mean that we can't check the typos with local checks?

Reply:
I would suggest, as we have done for coverage, to users to personally install the software that they will run locally, in this way users can choose the desidered version and the correct operating system. In this way `cargo xtask` tasks remain cross-platform.

Reply:
But we are already using Cargo to install typos-cli, so it is cross-platform right now! Maybe we can only install it locally but use the system version when the CI flag is activated?

Reply:
Yes, but there are many options to be considered when installing a binary on different operating systems, but no problem, it should be fixed now